tutorial/index.md

```diff
@@ -25,7 +25,7 @@ Before beginning, you must:
 
 ## Download the Source Code
 
-Download the [source code](https://github.com/aws-samples/aws-mobile-ios-notes-tutorial/archive/master.zip) from the GitHub repository.
+Download the [source code](https://github.com/ldomb/aws-mobile-ios-notes-tutorial/archive/master.zip) from the GitHub repository.
```

Comment:
should point to `aws-samples` or make the link relative

Reply:
fixed it in the new pull request